CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vue-wallet ChangeLog
 
+## 23.1.0 - 2023-11-xx
+
+### Added
```

Comment:
move into 23.0.0

Reply:
This can actually now be added as 23.1, I'll be releasing 23.0 shortly.